wordpress-mobile/gutenberg-mobile
# [Android] Fix for new line added to previous block on Enter.Key pressed

This PR fixes #392, by upgrading Aztec to the version that has the fix in it (https://github.com/wordpress-mobile/AztecEditor-Android/pull/768), and does make sure the  `onEnter` listener is called with the correct text (before inserting the new character into the editor).

Testing Steps:

- Tap enter in a para block
- A new block is inserted below
- Enter some text in the new block
- The block above should not change its content

Make sure to test this on Devices/Emulators that trigger both the software keyboard listeners (TextWatcher), and the HW listeners. On my Samsung S9 the keyboard is recognized as HW, and the key-listener path is followed, on my Nexus 5x instead, the keyboard is seen as software keyboard and TextWatcher(s) take place in the detection of input.

Aztec PR: https://github.com/wordpress-mobile/react-native-aztec/pull/101
